### ISSUE TITLE
Compatibility problems between the DSE and my new SSD hard disk

### ISSUE THREAD

**Issue by VasilisSkordilis** [adririquelme/DSE#6]
Hello,

My name is Vasilis and I am a Geology student in the University of Athens. I am currently working on my Thesis based on the application of this programme to my research interest. Through my supervisor's guidance I have downloaded the DSE software via DSE 2.XX 64bits installer version that is indicated in your official page. The software was functioning very well with the previous version of my pc attributes.

However, some days ago I changed my hard disk to Samsung SSD 970 EVO Plus  NVMe M.2 and I had a format in my pc. From this time on, I was unable to run the DSE programme although i have followed the exactly same procedure as before. I also tried to download the software via the alternative way proposed (Source code from the Github website. It can be executed running MATLAB under Windows or Linux. To start running the program, download all the files to your computer and execute the file DSE_v2XX.m under your MATLAB environment) but again I faced issues in running MATLAB.

To sum up, it would be really helpful if you could pinpoint any possible mistake that i am doing or any probable incompatibility of DSE to my new SSD.

Thank you very much in advance.
Kind regards, 
VS

**Comment by adririquelme** [comment]
The easyh way is to install Matlab 2020a and run this app: https://github.com/adririquelme/DSE/blob/master/DSE_v300.mlapp

I hope this works
